### PR TITLE
London Class 8 - Davood Khoshnood - JavaScript - Core 1 - Week 1

### PR DESCRIPTION
**Pull request by DavoodKhoshnood**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Davood Khoshnood
- Your City: London
- Your Slack Name: Davood

# Homework Details

- Module: JavaScript
- Week: 1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered exercises/B-hello-world/README.md](https://github.com/DavoodKhoshnood/JavaScript-Core-1-Coursework-Week1/blob/master/exercises/B-hello-world/README.md)

### CODE REVIEW

**Review by teniolao — CHANGES_REQUESTED** [review]

File: exercises/L-functions-nested/exercise.js [hunk 1]
```diff
@@ -1,5 +1,19 @@
+function greeting(name) {
+    return 'HELLO ' + name.toUpperCase();
+}
+
+
 var mentor1 = "Daniel";
```
[the remark this answers is not in the document]
Comment (DavoodKhoshnood) [reply exercises/L-functions-nested/exercise.js:6]: Hi Teniolao, I've changed and committed it again. thanks.

File: mandatory/1-syntax-errors.js [hunk 2]
```diff
@@ -25,16 +25,16 @@ To run the tests for just this one file, type `npm test -- --testPathPattern 1-s
...
+    'Hello, my name is Sonjide and I am 27 years old',
+  )
+})
 
-test("getTotal returns a string describing the total", () => {
-  expect(getTotal(23, 5)).toEqual("The total is 28");
-});
+test('getTotal returns a string describing the total', () => {
+  expect(getTotal(23, 5)).toEqual('The total is 28')
+})
```
Comment (DiegoJunior0) [inline mandatory/1-syntax-errors.js:40]: I assume you didn't mean to change all these tests, I assume from a change all? Be careful when doing this when tests are in the same file (though in the real world that wouldn't be case!)

**Comment by DiegoJunior0** [comment]
Excellent work Davood!

**Comment by DavoodKhoshnood** [comment]
> Excellent work Davood!
Thanks Henry!

**Pull request closed without merging**